.ci/build.groovy

```diff
@@ -9,15 +9,20 @@ def call(ghprbActualCommit, ghprbPullId, ghprbPullTitle, ghprbPullLink, ghprbPul
                         rm -rf /maven/.m2/repository/*
                         rm -rf /maven/.m2/settings.xml
                         rm -rf ~/.m2/settings.xml
+                        archive_url=http://fileserver.pingcap.net/download/builds/pingcap/client-java/cache/tikv-client-java-m2-cache-latest.tar.gz
```

Comment:
Since we are downloading latest tikv-client-java, can we remove .ci/build-client-java.sh?

Reply:
LGTM

Reply:
`tikv-client-java-m2-cache-latest.tar.gz` is not java client. It the m2 cache.

Reply:
The `maven cache` may be inaccurate. It's better to compile by `.ci/build-client-java.sh`.

Reply:
Sure